.maestro/testsV2/subFolder/profil/AccessibilitySection.yml

```
@@ -10,32 +10,63 @@ appId: ${MAESTRO_APP_ID}
 - assertVisible: 'Les engagements du pass Culture'
 - assertVisible:
     id: 'AccessibilityEngagement'
+
+- runFlow: # Permet de retourner sur la page précédente (cloud)
+    when:
+      true: ${MAESTRO_RUN_CLOUD_COMMANDS}
+    file: ../../subFolder/cloud/commands/TapOnBack.yml
+
 - tapOn:
     id: 'com.android.systemui:id/back'
+    optional: true
```

à ce niveau, on pourrait appeler un unique flow pour revenir en arrière, quelque soit la plateforme, l'intention est la meme

```suggestion
- runFlow:
    file: ../../subFolder/cloud/commands/TapOnBack.yml
```

voir message suivant ⬇️